src/Nox.Types/Types/DateTime/DateTime.cs

```diff
@@ -2,22 +2,22 @@
 using System.Globalization;
 
 namespace Nox.Types;
-public sealed class DateTime : ValueObject<System.DateTime, DateTime>
+public sealed class DateTime : ValueObject<DateTimeOffset, DateTime>
 {
     private DateTimeTypeOptions _dateTimeTypeOptions = new();
 
-    public DateTime() { Value = System.DateTime.MinValue; }
+    public DateTime() { Value = DateTimeOffset.MinValue; }
 
-    /// <inheritdoc cref="From(System.DateTime,Nox.Types.DateTimeTypeOptions)"/>
-    public new static DateTime From(System.DateTime dateTime) => From(dateTime, new DateTimeTypeOptions());
+    /// <inheritdoc cref="From(DateTimeOffset ,Nox.Types.DateTimeTypeOptions)"/>
+    public new static DateTime From(DateTimeOffset dateTime) => From(dateTime, new DateTimeTypeOptions());
 
     /// <summary>
     /// Creates and validates a new instance of <see cref="DateTime"/> from parsed value of <paramref name="dateTime"/>.
     /// </summary>
     /// <param name="dateTime"></param>
     /// <param name="options"></param>
     /// <returns>New <see cref="DateTime"/> object from parsed value of <paramref name="dateTime"/>.</returns>
-    public static DateTime From(System.DateTime dateTime, DateTimeTypeOptions options)
+    public static DateTime From(DateTimeOffset dateTime, DateTimeTypeOptions options)
```

Comment:
I think we should have another `From` method that still receives `System.DateTime` as a parameter.

If I'm not mistaken the difference it that `DateTimeOffset` includes information about timezone and `System.DateTime` doesn't, right? 
If that's the case the new method should create a DateTimeOffset with Utc time. And if you're doing this, don't forget to mention it in the xml comment.

Reply:
We could also have a `From` method that receives `System.DateTime` and timezone information as second parameter.

Reply:
@danield-ap done